PerfTools/AllocMonitor/README.md

```diff
@@ -114,4 +114,21 @@ service in the configuration. The parameters are
 
 The beginning of the file contains a description of the structure and contents of the file.
 
-This service is multi-thread safe.
\ No newline at end of file
+This service is multi-thread safe.
+
+
+### ModuleEventAllocMonitor
```

Comment:
I think it would be good to mention the `edmModuleEventAllocMonitorAnalyze.py` script here as well, maybe even with one or more examples of what kind if analyses the script can do.

Reply:
done